code/modules/fishing/fishing_equipment.dm

```diff
@@ -12,8 +12,8 @@
 	icon = 'icons/obj/fishing.dmi'
 	icon_state = "reel_blue"
 	w_class = WEIGHT_CLASS_SMALL
-	///A list of traits that this fishing line has, checked by fish traits and the minigame.
-	var/list/fishing_line_traits
+	///A bitflag of traits that this fishing line has, checked by fish traits and the minigame.
```

Comment:
bitfield no?

Reply:
looking it up it seems that bitflag is the correct terminology whereas bitfield is the term for things like packed integers (like storing four bytes in uint so you can represent rgba(255, 255, 255, 255) or whatever). i'll change it anyway though

Reply:
`bitflag` is singular though, isn't it? It's a single flag, not a field of flags? That's how I'd use the term, anyways. I'm not sure what to call the bitflag holder either though.